airflow/models/taskinstance.py

```diff
@@ -282,6 +296,18 @@ class TaskInstance(Base, LoggingMixin):  # pylint: disable=R0902,R0904
     executor_config = Column(PickleType(pickler=dill))
 
     external_executor_id = Column(String(ID_LEN, **COLLATION_ARGS))
+
+    # The trigger to resume on if we are in state DEFERRED
+    trigger_id = Column(BigInteger)
+
+    # Optional timeout datetime for the trigger (past this, we'll fail)
+    trigger_timeout = Column(UtcDateTime)
+
+    # The method to call next, and any extra arguments to pass to it.
+    # Usually used when resuming from DEFERRED.
+    next_method = Column(String(1000))
```

Comment:
```suggestion
    next_method = Column(String(1000, **COLLATION_ARGS))
```

(And similarly in the migration too I think?)

Reply:
Why are there some `Strings()` without `COLLATION_ARGS` then? Looking at it, I presumed they were only for the PKs.

Reply:
Oh could be that then. I have (thankfully) forgotten much about MySQL. It probably doesn't matter here as this will be limited to ASCII chars only, right? (Cos this is a python class/function name)

Reply:
Exactly, I don't think it matters either way, I was just pattern-matching to what was done there already. It might be something to do with indexes?

Reply:
Oh yes, likely is that. Something to do with UTF8 3 vs 4 byte chars and maximum size of a index column in MySQL. Not an issue here.